optaplanner-benchmark/src/main/java/org/optaplanner/benchmark/impl/result/BenchmarkResultIO.java

```diff
@@ -108,9 +99,6 @@ protected PlannerBenchmarkResult readPlannerBenchmarkResult(File plannerBenchmar
             String benchmarkReportDirectoryName = plannerBenchmarkResultFile.getParentFile().getName();
             plannerBenchmarkResult = PlannerBenchmarkResult.createUnmarshallingFailedResult(
                     benchmarkReportDirectoryName);
-        } catch (XStreamException e) {
```

Comment:
Interesting. I was assuming it does, because why else would IDEA warn me about it? Anyway, I'll put it back.